.github/scripts/fbgemm_gpu_build.bash

```diff
@@ -444,6 +444,24 @@ build_fbgemm_gpu_develop () {
   # canceled for going over ulimits
   echo "[BUILD] Building (develop) FBGEMM-GPU (VARIANT=${fbgemm_variant}) ..."
   # shellcheck disable=SC2086
+  ver=$(apt show rocm-libs | grep Version | grep -Po '(\d+\.\d+\.\d+)')
+  if [ "$ver" == "5.6.0" ]; then
+    new_path="$PATH:/opt/rocm/llvm/bin:\
+/opt/rocm/opencl/bin:\
+/opt/rocm/hip/bin:\
+/opt/rocm/hcc/bin:\
+/opt/rocm/bin:\
+/opt/conda/bin:\
+/usr/local/sbin:\
+/usr/local/bin:\
+/usr/sbin:\
+/usr/bin:\
+/sbin:\
+/bin"
+    echo $new_path
+    print_exec conda env config vars set ${env_prefix} PATH="$new_path"
+    (exec_with_retries 3 conda run ${env_prefix} echo "===============$PATH") || return 1
+  fi
```

Comment:
You may want to define this configuration inside `__configure_fbgemm_gpu_build_rocm` instead